packages/trader/src/AppV2/Components/DatePicker/date-picker.tsx

```diff
@@ -6,24 +6,40 @@ import { Localize } from '@deriv/translations';
 import { DEFAULT_DATE_FORMATTING_CONFIG } from 'AppV2/Utils/positions-utils';
 
 type TDateRangePicker = {
-    handleDateChange: (values: { to?: moment.Moment; from?: moment.Moment; is_batch?: boolean }) => void;
-    isOpen?: boolean;
+    applyHandler: () => void;
```

Comment:
The difference between `onClose` and `applyHandler`: `onClose` will close only DatePicker Action sheet, but not Action sheet with all time filters; `applyHandler` will closing both of them